src/rutabaga_core.rs

```diff
@@ -1363,6 +1363,12 @@ impl RutabagaBuilder {
         self
     }
 
+    /// Sets virglrenderer flags directly from a u32.
+    pub fn set_virglrenderer_flags(mut self, flags: VirglRendererFlags) -> RutabagaBuilder {
```

Comment:
I wonder if you need this if you adopt the newer rutabaga API.

For example, what `libkrun` does is:

```
    uint32_t virgl_flags = VIRGLRENDERER_USE_EGL | VIRGLRENDERER_DRM |
            VIRGLRENDERER_THREAD_SYNC | VIRGLRENDERER_USE_ASYNC_FENCE_CB;
```

This is the config rutabaga will choose if you do:

```
capset_mask = calculate_capset_mask(":virgl2:drm")
RutabagaBuilder::new(capset_mask, fence_handler).builds()
```
This is because virgl is only supported with EGL now (no GLX option), and only the fence callback is allowed with rutabaga anyways.  So maybe it would be to have libkrun side CL, that works with the WIP branch, so we can see how both sides fit together?